aws/sdk-codegen/src/test/kotlin/software/amazon/smithy/rustsdk/TestUtil.kt

```diff
@@ -51,7 +51,7 @@ fun awsSdkIntegrationTest(
 
 fun awsIntegrationTestParams() =
     IntegrationTestParams(
-        cargoCommand = "cargo test --features test-util behavior-version-latest",
+        cargoCommand = "cargo test --features test-util,behavior-version-latest --tests --lib",
```

Comment:
Are the `--tests --lib` args actually doing anything?

Reply:
yes! they mean we don't run doc tests which changes the test run for 30 seconds to one second